functionaltests/src/test/java/util/CustomWebDriver.java

```diff
@@ -14,7 +14,7 @@
     private ChromeDriver driver;
 
     CustomWebDriver() {
-        String currentDriver = SystemUtils.IS_OS_MAC ? "chromedriver_ios_2.40" : "chromedriver_unix_2.40";
+        String currentDriver = SystemUtils.IS_OS_MAC ? "chromedriver_mac_75" : "chromedriver_linux_75";
```

Comment:
Why not latest version which is 76.0.3809.68 right now? See: http://chromedriver.chromium.org/downloads

Reply:
Because it's not stable version, but only beta. And we're testing with stable version. https://chromereleases.googleblog.com/